packages/contracts-bedrock/snapshots/abi/SoulGasToken.json

```diff
@@ -8,7 +8,8 @@
       }
     ],
     "stateMutability": "nonpayable",
-    "type": "constructor"
+    "type": "
```

Comment:
Looks like this change breaks `snapshots-check-no-build`

Reply:
Does the recent op-es commit pass the `snapshots-check-no-build`? I just accepted `SoulGasToken.json` file in `op-es`. I'll also double check that.